<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <listeners>
    <listener class-name="Academy.TestComponents.Listeners02"/>
    <listener class-name="Academy.TestComponents.TestRetryAnalyzerListener"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Submit Order Test(failed)" verbose="0">
    <classes>
      <class name="Academy.SeleniumFrameworkAgain0901.StandAloneTest171">
        <methods>
          <include name="closeBrowser"/>
          <include name="ordersHistoryValidation"/>
          <include name="submitOrder"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- Academy.SeleniumFrameworkAgain0901.StandAloneTest171 -->
    </classes>
  </test> <!-- Submit Order Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
